.gitignore

```diff
@@ -16,6 +16,8 @@ node_modules/
 .site/
 _vvv/
 .idea/
+assets/js/onboarding-wizard.js.LICENSE.txt
```

Comment:
Could you make these globally ignored instead of specific to `onboarding-wizard`?